### PR TITLE
React OWIN article to ASP.NET Core rename

### PR DESCRIPTION
**Pull request by MaherJendoubi**

### CODE REVIEW

File: aspnet/fundamentals/owin.rst [hunk 1]
```diff
@@ -83,10 +83,10 @@ With this in place, all that's required to run an ASP.NET application using this
 
 When run, this command (equivalent to running ``dnx web`` from a command line) will search for a package called "NowinSample" that contains an implementation of ``IServerFactory``. If it finds one, it will initialize and start the server as detailed above. Learn more about the built-in ASP.NET :doc:`/fundamentals/servers`.
 
-Run ASP.NET 5 on an OWIN-based server and use its WebSockets support
+Run ASP.NET Core on an OWIN-based server and use its WebSockets support
 --------------------------------------------------------------------
```
Comment (danroth27) [inline aspnet/fundamentals/owin.rst]: The underline needs to be the full length of the title

**Comment by danroth27** [comment]
@MaherJendoubi Thank you for this contribution! Please note that we are waiting to roll out the ASP.NET Core rename in our docs until we have shipped updated bits with the new names (see https://blogs.msdn.microsoft.com/webdev/2016/02/01/an-update-on-asp-net-core-and-net-core/). We are staging all of the rename work in our dev branch. Could you please update this PR to be based on dev?

**Comment by MaherJendoubi** [comment]
@danroth27 done.

**Comment by danroth27** [comment]
@MaherJendoubi I don't see a new commit on this PR and it's still based on master. Did you forget to push?

**Comment by MaherJendoubi** [comment]
@danroth27 Is it Ok now? https://github.com/aspnet/Docs/pull/979

**Comment by MaherJendoubi** [comment]
@danroth27 do you see please my PR on dev branch?

**Comment by danroth27** [comment]
@MaherJendoubi Yup, that's why I closed this one.

**Pull request closed without merging**